platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -1082,4 +1092,37 @@ public void computeHash() {
             Thread.currentThread().interrupt();
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void createSnapshot(Path targetPath) {
+        throwIfMutable();
+        throwIfDestroyed();
+        try {
+            writeMerkleRootToFile(targetPath, this);
+        } catch (final Throwable e) {
+            logger.error(
+                    EXCEPTION.getMarker(),
+                    "Unable to write signed state to disk for round {} to {}.",
+                    getReadablePlatformState().getRound(),
+                    targetPath,
+                    e);
+        }
+    }
+
+    private static void writeMerkleRootToFile(final Path directory, final MerkleRoot merkleRoot) throws IOException {
+        writeAndFlush(
+                directory.resolve(SIGNED_STATE_FILE_NAME), out -> writeMerkleRootToStream(out, directory, merkleRoot));
+    }
+
+    private static void writeMerkleRootToStream(
+            final MerkleDataOutputStream out, final Path directory, final MerkleRoot merkleRoot) throws IOException {
+        out.write(VERSIONED_FILE_BYTE);
+        out.writeInt(FILE_VERSION);
```

Comment:
this is the file version that should be changed

Reply:
Done